src/database/initialization/changemaker_to_json.sql

```diff
@@ -7,7 +7,19 @@ CREATE FUNCTION changemaker_to_json(
 RETURNS jsonb AS $$
 DECLARE
   proposal_field_values_json JSONB;
+  fiscal_sponsors_json JSONB;
 BEGIN
+  -- Terminate changemaker recursion by passing null here. This also means no deep fields on sponsors.
+  SELECT jsonb_agg(changemaker_to_json(fiscal_sponsors.*, NULL))
+  INTO fiscal_sponsors_json
+  FROM changemakers AS fiscal_sponsors
+  INNER JOIN fiscal_sponsorships fs
+    ON fs.fiscal_sponsor_changemaker_id = fiscal_sponsors.id
+    AND fs.fiscal_sponsee_changemaker_id = changemaker.id
+    AND (fs.not_after IS NULL or fs.not_after > now())
```

Comment:
Use the SQL helper function we have `is_expired(fs.not_after)`

so `NOT is_expired...`

Reply:
After seeing your PR a moment ago, I thought you might mention it here, you're right!

Reply:
Nice catch, thanks, fixed.